setup.py

```diff
@@ -32,8 +32,6 @@
       description='Application for synchronizing customer directories with the Adobe Enterprise Admin Console',
       classifiers=[
           'Development Status :: 5 - Production/Stable',
-          'Programming Language :: Python :: 2.7',
-          'Programming Language :: Python :: 3.4',
           'Programming Language :: Python :: 3.5',
```

Comment:
While we're at it can we remove the Python 3.5 tag? Technically it doesn't do anything because we don't publish this project to PyPI, but it is nice to keep it updated for the benefit of developers to hint at which Python versions we support.  Our CI builds all target 3.6 only in the 3.X series.